client/src/Component/Button/index.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
```

Comment:
I think this folder have to move to a common folder

Reply:
+1